README.md

```diff
@@ -782,3 +794,6 @@ variant of the Isearch interface that takes inspiration from the
 standard text search interface found in almost every other modern
 piece of software, such as web browsers. The result is
 [CTRLF](https://github.com/raxod502/ctrlf).
+
+Recently the [Consult](https://github.com/minad/consult) package has
```

Comment:
Maybe better start with "There is also the ..."? To keep it more timeless so to speak.

Reply:
Ok!